src/sentry/utils/sdk_crashes/cocoa_sdk_crash_detector.py

```diff
@@ -12,7 +12,11 @@ def is_sdk_crash(self, frames: Sequence[Mapping[str, Any]]) -> bool:
         if not frames:
             return False
 
-        for frame in frames:
+        # The frames are ordered from caller to callee, or oldest to youngest.
+        # The last frame is the one creating the exception.
+        # Therefore, we must iterate in reverse order.
+        frames_reversed = frames[::-1]
+        for frame in frames_reversed:
```

Comment:
```suggestion
        for frame in reversed(frames):
```

Today I learned about that slice syntax. Though it is a bit confusing and too much magic. A `reversed` iterator should also avoid creating a copy and just iterate in the desired order.